demo/src/components/CombinedFilters.js

```diff
@@ -13,8 +13,8 @@ const CombinedFilters = () => {
     return (
         <CombinedFacets
             transform={transform}
-            RangeFacetItemComponent={RangeFacetItemComponent}
-            TextFacetItemComponent={TextFacetItemComponent}
+            RangeFacetItemComponent={<RangeFacetItemComponent />}
```

Comment:
rename the prop to denote that the value is an instance. `RangeFacetItemComponent` -> `rangeFacetItemComponent `

Reply:
done